bbn-test-5/babylon-node/README.md

```diff
@@ -0,0 +1,283 @@
+        -X "github.com/cosmos/cosmos-sdk/version.BuildTags=netgo,ledger" \
+        -w -s' \
+    -trimpath ./...
+```
+
+Now it has successfully compiled, lets check the available actions
+through the `babylond` command:
+
+```shell
+babylond
+```
+
+Which will give us a list of available actions.
+
+```shell
+Available Commands:
+  add-genesis-account Add a genesis account to genesis.json
+  collect-gentxs      Collect genesis txs and output a genesis.json file
+  comet               CometBFT subcommands
+  config              Utilities for managing application configuration
+  ...
+```
+
+If the babylond command isn't found, ensure `$GOPATH/bin` is in your shell's 
+`$PATH`. Add it with:
+
+ ```shell 
+ export PATH=$HOME/go/bin:$PATHecho 'export PATH=$HOME/go/bin:$PATH' >>
+ ~/.profile 
+ ```
+
+## Setup your node, home directory and configure 
+
+Initialize your node and create the necessary configuration directory. 
+This command will generate several important configuration files 
+including `app.toml`, `client.toml`, and `genesis.json`:
+
+```shell
+babylond init <moniker> --chain-id bbn-test-5 --home <path>
+```
+
+The `<moniker>` is a unique identifier for your node. So for example `node0`.
+The `<path>` is the directory where the node files will be stored 
+(e.g. `--home ./nodeDir`).
+
+After initialization, you'll need to modify two important configuration files:
+
+1. First, open `app.toml` and update these essential settings:
+
+```shell
+Base configuration
+minimum-gas-prices = "0.005ubbn"
+iavl-cache-size = 0
+iavl-disable-fastnode=true
+
+[btc-config]
+
+# Configures which bitcoin network should be used for checkpointing
+# valid values are: [mainnet, testnet, simnet, signet, regtest]
+network = "signet" # Bitcoin network for checkpointing
+```
+
+In `app.toml` under `[btc-network]`change network to `signet` and under 
+`Base Configuration` set `iavl-cache-size = 0` and `iavl-disable-fastnode=true` 
+instead of what is listed in the automatically generated template. 
+Additionally, add `minimum-gas-prices = "0.005ubbn"`
+
+Navigate to `config.toml`. Add in your seed that should look something like 
+this `8fa2d1ab10dfd99a51703ba760f0ef555ae88f36@16.162.207.201:26656`
+
+```shell
+ P2P Configuration Options    
+
+# Comma separated list of seed nodes to connect to
+seeds = "8fa2d1ab10dfd99a51703ba760f0ef555ae88f36@16.162.207.201:26656"
+```
+
+Next, you'll need to obtain the network's genesis file. This file contains 
+the initial state of the blockchain and is crucial for starting from the 
+correct point. You have two options:
+
+1. Download directly from the RPC endpoint: `https://rpc.devnet.babylonlabs.io/genesis`
+
+2. Or use these commands in your terminal:
+
+```shell
+wget https://github.com/babylonlabs-io/networks/raw/main/bbn-test-5/genesis.tar.bz2 # TODO: update this file name if necessary
+tar -xjf genesis.tar.bz2 && rm genesis.tar.bz2
+mv genesis.json ~/.babylond/config/genesis.json
```

Comment:
Note that `~/.babylond` might not be the home directory being used by the reader.